### PR TITLE
OOP School Library: TDD

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

* Added unit tests using Rspec.
* Created Solver class using TDD.

## Testing

`rspec spec --format doc`

## Created by:

@alonsofl & @smunozmo 

## Notes:

✅  - Used  Ruby.
✅  - Used  Rubocop.
✅  - No linters errors.

### CODE REVIEW

File: spec/solver_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      fizzbuzz = solver.fizzbuzz(5)
+      expect(fizzbuzz).to eq 'buzz'
+    end
+
+    it 'Should return fizzbuzz' do
+      solver = Solver.new
+      fizzbuzz = solver.fizzbuzz(15)
+      expect(fizzbuzz).to eq 'fizzbuzz'
+    end
+  end
```
Comment (oxenprogrammer) [inline spec/solver_spec.rb:62]: **Required Changes**
Good effort so far with the test. You have handled almost all edge cases. 
- [x] I would love you to also handle the `edge case` where `# N is not divisible by 3 or 5`  i.e test using 7 for the fizzbuzz function. Let us try and write one more test case.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @smunozmo and @alonsofl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
- [ ] Please update your README file with instructions of how to setup the repository locally and as well as run the tests locally.
_Check the comments under the review for more required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Thank you! @oxenprogrammer @vichuge
